profiles/debug.lua

```diff
@@ -0,0 +1,99 @@
+-- Enable calling our lua profile code directly from the lua command line,
```

Comment:
This is actually a good idea. We could do a real mock of the C++ interface in lua for testing.